### PR TITLE
Fixes from checklist review related to #1323

### PR DESCRIPTION
**Pull request by reshmakh**

### CODE REVIEW

**Review by codyebberson — APPROVED** [review]

File: packages/docs/docs/auth/smart-scopes.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# SMART Scopes for Access Control
+
+SMART on FHIR’s authorization scheme uses OAuth scopes to communicate (and negotiate) access requirements. There is relevant [background information on HL7.org](http://hl7.org/fhir/smart-app-launch/STU2/scopes-and-launch-context.html).
```
Comment (rahul1) [inline packages/docs/docs/auth/smart-scopes.md:3]: I think backing up a few steps and giving some more framing would help make this document more accessible. To me, the biggest thing that's missing is information about "when do I need to use this?". Some questions that would be helpful to answer:

- What is the overall purpose of SMART on FHIR scopes? Might be able to copy some verbiage from [here](http://hl7.org/fhir/smart-app-launch/STU2/app-launch.html)
- What kind of apps will _need_ to use this? Some examples: 
  -  Any 1st party app that Medplum customer builds?
  - An app that launches from inside the Medplum App?
  - An app that I (medplum customer) am building to launch from a 3rd party EPIC instance? 
  - Allowing 3rd party developers to build apps on top of my (medplum customer's) data?

File: packages/docs/docs/auth/smart-scopes.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+Medplum supports the `SMART App Launch 2.0.0` [standard](https://hl7.org/fhir/smart-app-launch/).
+
+Historically, apps developed with SMART Scopes were designed to be links, embedded in a traditional EHR. Once those links were clicked on an app would launch and authenticate the user, and request a certain FHIR resources, or grant rights like `offline_access`. Commonly, EHRs support "launching" from either a patient history screen or an encounter documentation screen. So when you see "launch" language in the documentation, hopefully that gives some context on what that means.
```
Comment (rahul1) [inline packages/docs/docs/auth/smart-scopes.md:7]: I think this paragraph might be too confusing. 
What I found confusing was that this paragraph starts off saying "Historically, SMART scopes were designed for X", which implies that this is _not_ what SMART scopes are used for nowadays. But it doesn't seem to clearly address what use cases SMART scopes _are_ used for nowadays

After addressing my comment above, this probably becomes unnecessary.

File: packages/docs/docs/auth/smart-scopes.md (hunk 1 above)
Comment (rahul1) [inline packages/docs/docs/auth/smart-scopes.md:3]: Maybe link the words "Oauth scopes" to some basic supporting info about what Oauth scopes are? I found the Auth0 docs pretty well written: https://auth0.com/docs/get-started/apis/scopes

File: packages/docs/docs/auth/smart-scopes.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+## Supported Scopes
+
+Medplum supports the following scopes.
+
+- `patient/\*.rs`
```
Comment (codyebberson) [inline packages/docs/docs/auth/smart-scopes.md:13]: I'd remove the backslash before the asterisk here.  It should just be `patient/*.rs`

File: packages/docs/docs/auth/smart-scopes.md (hunk 1 above)
Comment (codyebberson) [reply packages/docs/docs/auth/smart-scopes.md:3]: Yeah... This page is the best explanation: https://hl7.org/fhir/smart-app-launch/scopes-and-launch-context.html

We should either reference that page directly, or mimic the flow.

**Pull request merged**